service-api/module/Application/config/module.config.php

```diff
@@ -246,6 +246,36 @@
                     ],
                 ],
             ],
+            'search_address_by_postcode' => [
+                'type'    => Segment::class,
+                'options' => [
+                    'route'    => '/cases/:uuid/search-address-by-postcode/:postcode',
+                    'defaults' => [
+                        'controller' => Controller\IdentityController::class,
+                        'action'     => 'searchAddressByPostcode',
+                    ],
+                ],
+            ],
```

Comment:
You should be able to get this from the Sirius endpoint I added yesterday, rather than using the Paper ID API.